src/pages/cli/cm/validate.md

```
@@ -0,0 +1,36 @@
+```bash
+# Validate cache in current candy machine directory
+mplx cm validate
+
+# Validate specific cache file
+mplx cm validate <path_to_asset_cache>
+
+# Validate on-chain insertions (requires candy machine to exist)
+mplx cm validate --onchain
+```
+
+If the validation command shows issues, depending on the error you might want to check the issues with your assets or run the upload or insert command.   
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 3

(MD009, no-trailing-spaces)